.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and Deploy docs
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Conda
+      uses: s-weigand/setup-conda@v1
+      with:
+        activate-conda: false
+        conda-channels: conda-forge
+
+    - name: Create environment
+      shell: bash -l {0}
+      run: |
+        conda create --name TEST python=3 python-build cython numpy --file requirements-dev.txt
```

Comment:
We don't need cython.

Reply:
Argh. That "copy-n-paste" error was there for a while. I removed it from everywhere now. Luckily it did not cause any side effect on the builds.